uiowa/uiowa
# Joe's fantastic pull of fixes

# To Test Taxonomy Term RSS

(fixes #5229)
- Pull a site that has tagged content (articles, pages, people). View the RSS feed, preferably with twig debug turned off (false, docroot/sites/blt.development.services.yml). See that there aren't duplicate titles/dates, breadcrumbs, etc. Description should be filled if field_teaser is filled.

<img width="862" alt="Screen Shot 2022-06-10 at 10 52 02 AM" src="https://user-images.githubusercontent.com/4663676/173104126-b2a67dfc-671d-4986-8e89-2cb92795cba1.png">

# To Test imagecache external configuration change

(fixes #5153)
- Go to the status report page of any pulled down site. See that imagecache external is "properly configured" and that you are able to add an events or aggregator block and have images loaded.

<img width="605" alt="Screen Shot 2022-06-09 at 4 57 33 PM" src="https://user-images.githubusercontent.com/4663676/173103366-e66a0a0b-c34d-445b-9c88-cc8b108ffb26.png">

# To Test imagecache external clean up patch

(fixes #5150)
- This is hard to test unless you have a local site with several files/externals. The idea though is that you pull this site down, and run cron. that directory along with it's respective styles directory should clean up some if not all the images there and you should be able to revisit an events or aggregator block with images and have those folders fill up again. This doesn't run every cron however and a state timestamp is set. You can try adjusting the frequency temporarily, manually adjusting the timestamp in the db or you might have to trust me on this one.

Before:
<img width="499" alt="Screen Shot 2022-06-10 at 10 44 17 AM" src="https://user-images.githubusercontent.com/4663676/173103237-d4f2fc51-9055-432f-b8d7-dcb55efc638b.png">

After:
<img width="513" alt="Screen Shot 2022-06-10 at 10 47 48 AM" src="https://user-images.githubusercontent.com/4663676/173103233-1a52e453-9241-49c8-a697-2a64e1c8e9b0.png">

# To Test Image block title attribute

(fixes #5156)
- Add an image block with link text. View the published page. Hover over the image and see that the title tooltip appears with the text.

<img width="372" alt="Screen Shot 2022-06-10 at 10 50 44 AM" src="https://user-images.githubusercontent.com/4663676/173103774-1a362b2c-4510-4058-b07e-0b98ab2e1938.png">

# To Test Events teaser canceled

(fixes #4070)
- Create two events blocks:
  - One that brings up a canceled event: 
    - https://default.uiowa.ddev.site/event/77436/0
    - https://content.uiowa.edu/neuroscience-pharmacology-seminar-elizabeth-heller-phd
    - unit: pharmacology
    - interest: Research
    - type: Seminar
  - One that just brings up general upcoming events. 
- Notice that https://content.uiowa.edu/neuroscience-pharmacology-seminar-elizabeth-heller-phd shows up with a canceled badge. This is the same for v2. Note that the date, virtual, location, description of the other events is unaffected.

<img width="760" alt="Screen Shot 2022-06-10 at 11 57 25 AM" src="https://user-images.githubusercontent.com/4663676/173115575-e1dac446-d43c-4935-a7d2-7bd7b15e4999.png">

# To test consistent tag widget

(fixes #4969)
- pull down a site and try adding different media types through the add media interface. they shouldn't be a single line field but the multi line option. help text describing comma-separated was removed. I didn't change the media library/browser versions because it was acting funky in the modal and putting the `add another item` button down with the `save and select` button.

<img width="462" alt="Screen Shot 2022-06-10 at 1 57 11 PM" src="https://user-images.githubusercontent.com/4663676/173133963-2066dfd4-9324-46e5-9727-30f33ac8fee1.png">

<img width="697" alt="Screen Shot 2022-06-10 at 1 58 54 PM" src="https://user-images.githubusercontent.com/4663676/173133952-2d909ce2-680a-4408-a99e-525afd613ece.png">
